pcdsdevices/lxe.py

```diff
@@ -210,3 +212,27 @@ def move(self, position, wait=True, timeout=None, moved_cb=None):
     def wm(self) -> float:
         # Remove the PseudoPosition tuple for FltMvInterface compatibility
         return super().wm[0]
+
+
+class LaserEnergyTimingLxt(PVPositioner, FltMvInterface):
+    """
+    "lxt" motor, which may also have been referred to as Vitara.
+
+    Conversions for Vitara FS_TGT_TIME nanoseconds <-> seconds are done
+    internally, such that the user may work in units of seconds.
+    """
+
+    _fs_tgt_time = Cpt(EpicsSignal, ':VIT:FS_TGT_TIME', auto_monitor=True)
+    setpoint = Cpt(UnitConversionDerivedSignal,
+                   derived_from='_fs_tgt_time',
+                   derived_units='s',
+                   original_units='ns',
+                   )
+
+    # A motor (record) will be moved after the above record is touched, so
+    # use its done motion status:
+    done = Cpt(EpicsSignal, ':MMS:PH.DMOV', auto_monitor=True)
+    done_value = 1
+
+    def __init__(self, prefix='', *, egu=None, **kwargs):
+        super().__init__(prefix, egu='s', **kwargs)
```

Comment:
These two lines are kind of strange, the default prefix surely never works and the user's egu argument is ignored if passed.

Reply:
The `prefix=''` is a carryover from the superclass `__init__`.

I'll add a check for `egu` to complain if it's set, as I don't plan on having it customizable. Users appear to want to work with seconds here, so adding configuration is overkill.

Reply:
Resolved in 86b14de